refgenconf/refgenconf.py

```diff
@@ -2230,10 +2230,10 @@ def upgrade_config(target_version, filepath, force=False,
         cnt += 1
         try:
             get_json_url(server, 'index_v3__get')
```

Comment:
`index_v3__get` is an auto-generated operationID based on a function name that creates an API endpoint, so It could change in the future without notice and `upgrade_config` would break. 

Use an endpoint that has a custom operationID defined.